# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux/99-quiet.patch
# Copyright (C) 2010 - 2014 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We are not soo much overly graphic oriented distro, the more it's important
that our default text bootup, login, install etc. experience is not too noisy.
(This is like the kernel command line parameter "quiet", just that we do not
 have to tweak all boot loader configs to always set it.)

  - Rene Rebe <rene@exactcode.de>

--- linux-3.16/include/linux/printk.h.vanilla	2014-08-13 10:51:09.516000010 +0000
+++ linux-3.16/include/linux/printk.h	2014-08-13 10:53:02.596000011 +0000
@@ -37,7 +37,7 @@
 #define CONSOLE_LOGLEVEL_SILENT  0 /* Mum's the word */
 #define CONSOLE_LOGLEVEL_MIN	 1 /* Minimum loglevel we let people use */
 #define CONSOLE_LOGLEVEL_QUIET	 4 /* Shhh ..., when booted with "quiet" */
-#define CONSOLE_LOGLEVEL_DEFAULT 7 /* anything MORE serious than KERN_DEBUG */
+#define CONSOLE_LOGLEVEL_DEFAULT 4 /* "quiet" built-in, vanilla default is 7 */
 #define CONSOLE_LOGLEVEL_DEBUG	10 /* issue debug messages */
 #define CONSOLE_LOGLEVEL_MOTORMOUTH 15	/* You can't shut this one up */
 
